12-Monitoring/12-monitoring.md

```diff
@@ -0,0 +1,28 @@
+# Monitoring
+### TODO:
+- What is monitoring and why is it important?
```

Comment:
There is a chance that this chapter will be back in the branch training

Reply:
So should I delete it?